aqua.yaml

```
@@ -3,18 +3,18 @@
 # https://aquaproj.github.io/
 registries:
   - type: standard
-    ref: v4.444.0 # renovate: depName=aquaproj/aqua-registry
+    ref: v4.442.1 # renovate: depName=aquaproj/aqua-registry
```

```suggestion
    ref: v4.444.0 # renovate: depName=aquaproj/aqua-registry
```

Sorry if I misled you, but I just want this line left as it was. Could you please modify as above?

---

I merged https://github.com/cybozu-go/cattage/pull/146 this morning, so I updated it with v4.445.0. Is this understanding correct?

---

What we need to do is match the base branch on this line. This line on the current base branch is `v4.444.0 ` so we should leave this as `v4.444.0` I believe. It's correct if we find no diff on this line after committing.
Please note if you rebase, the base branch can change.